apps/console/src/features/identity-providers/data/identity-provider-templates/templates/saml-identity-provider/saml-idp-wizard-help.tsx

```diff
@@ -29,35 +29,34 @@ const SamlIDPWizardHelp: FunctionComponent<Props> = (props: Props): ReactElement
 
     return (
         <div data-testid={ testId }>
-            <Heading as="h5">Service provider entity id</Heading>
+            <Heading as="h5">Service provider entity ID</Heading>
             <p>
-                A globally unique name for IDPs under Asgardeo. This can be any value but when you configure a service
-                provider in the external IDP you should give the same value as the Service Provider Entity ID.
+                This value will be used as the <code>&lt;saml2:Issuer&gt;</code> in the SAML requests initiated from
+                Asgardeo to external Identity Provider (IdP). You need to provide a unique value as the service
```

Comment:
Get product name from config.

```suggestion
                Asgardeo to external Identity Provider (IdP). You need to provide a unique value as the service
```

Reply:
Addressed this in commit e245413